index.css

```diff
@@ -1,330 +1,318 @@
     align-items: center;
-    
+    padding: 15px 15%;
+    background-color: #fff;
 }
-.icon {
-    background-image: url(/images/Group\ 19.png);
-    width: 153px;
-    height: 51px;
-    cursor: pointer;
+
+.logo {
+    font-size: 20px;
+    font-weight: bold;
 }
-.login {
-    background: #3692FF;
+
+.login-btn {
     width: 128px;
     height: 48px;
     border-radius: 8px;
-    color: #fff;
-    display: flex;
-    justify-content: center; 
-    align-items: center;    
+    padding: 8px 16px;
+    background-color: #007bff;
+    color: white;
+    border: none;
     cursor: pointer;
 }
-.login:hover {
-    background: #1251AA;
-}
-nav {
+
+/* 히어로 섹션 */
+.cta-button {
     width: 100%;
-    height: 540px;
-    background: #cfe5ff;
-    display: flex;
-}
-nav .container {
-    width: calc(100% - 27vw);
-    margin: 0 auto;
-    display: grid;
-    grid-template: 1fr / 1fr  1fr;
-    align-items: end;
-    flex-direction: row;
-}
-.navDesc h4{
-    font-size: 1.4em;
-    color: #374151;
-}
-.navDesc button {
-    margin-bottom: 30%;
-    margin-top: 30px;
-    width: 20vw;
-    height: 46px;
-    color: #fff;
-    background: #3692FF;
+    display: inline-block;
+    padding: 10px 20px;
+    background-color: #007bff;
+    color: white;
+    text-decoration: none;
     border-radius: 40px;
-    border: none;
-}
-.navDesc button:hover {
-    background: #1251AA;
-}
-.navImgWrap {
-    width: 51vw;
-    height: 21vw;
+    text-align: center;
+
 }
-.navImg {
-    background-image: url(/images/Img_home_top.png);
+.hero {
+    display: flex;
+    align-items: center;
+    /* padding: 15px 15%; */
+    padding-top: 200px;
     width: 100%;
-    height: 100%;
-    background-repeat: no-repeat;
-    background-size: contain;
+    /* flex-direction: column; */
+    justify-content: center; /* 중앙 정렬 */
+    align-items: center;
+    background-color: #cce5ff;
+    height: auto; /* 높이 지정 (필요에 따라 조정) */
+    position: relative;
 }
 
-main {
-    width: 100%;
-    height: auto;
-    background: #fff;
-    padding-bottom: 10%;
-}
-main .container {
-    width: calc(100% - 32vw);
-    margin: 0 auto;
-    /* background: #000; */
-    height: auto;
-}
-.mainWrap {
-    width: 100%;
-    height: 444px;
+.hero-content {
+    width: 20%;
     display: flex;
-    background: #fcfcfc;
-    margin-top: 10%;
+    align-items: center;
+    padding-left: 15%;
 }
-.hotImg{
-    background-image: url(/images/Img_home_01.png);
-    width: 60%;
-    height: 444px;
-    background-repeat: no-repeat;
-    background-size: cover;
+
+.hero-text {
+    text-align: left;
+    margin-bottom: 20px; 
 }
-.searImg{
-    background-image: url(/images/Img_home_02.png);
-    width: 60%;
-    height: 444px;
+
+.hero .hero-image {
+    width: 100%; 
+    height: 340px;
+    background-image: url('/images/mainBtn1.png'); 
+    background-size: contain; 
     background-repeat: no-repeat;
-    background-size: cover;
-}
+    background-position: center; 
```

Comment:
❗️ 수정요청
현재 해당 이미지가 디자인대로 구현될 수 있게 이미지 위치를 bottom으로 바꿔주시고
.hero 요소의 justify-content 속성을 space-between으로 변경해주시는 것을 추천드려요~

```suggestion
    background-position: bottom; 
```

<img width="714" alt="스크린샷 2025-03-25 오후 4 12 14" src="https://github.com/user-attachments/assets/e75cd319-098d-4a78-8fb9-6a04fd74f174" />